week-3/quote-generator/quotes.js

```diff
@@ -491,3 +491,46 @@ const quotes = [
 ];
 
 // call pickFromArray with the quotes array to check you get a random quote
+
+const quote = document.getElementById("quote");
+const author = document.getElementById("author");
+const quoteBtn = document.getElementById("new-quote");
+
+const getRandomQuote = () => {
+  const pickedQuote = pickFromArray(quotes);
+  quote.innerText = pickedQuote.quote;
+  author.innerText = pickedQuote.author;
+};
+
+getRandomQuote();
+quoteBtn.addEventListener("click", getRandomQuote);
+
+// Auto Play
+
+const checkbox = document.querySelector("input[name=checkbox]");
+
+function autoPlay() {
+
+  let currentCheck;
```

Comment:
What would the difference be between if you put this line of code where it is, if you put it outside the `function autoPlay` (e.g. on line 511), and if you put it inside the `start` function (e.g. on line 518)?